### DISCUSSION TITLE
Using custom rate schedule in Reopt

### DISCUSSION THREAD

**Discussion by Fahad021** [NREL/REopt-API-Analysis#12] [8 - API: Application Programming Interface] [answered]
Hi @NLaws @jmpohl, 
is there a way we can input custom rate schedule, if the rate is not present in OpenEI rate database?

**Comment by NLaws** [comment, marked as answer]
Yes there are several options for custom tariff inputs, the most flexible of which is modifying an existing or creating your own `urdb_respsone` dictionary as an input (which mirrors the URDB API response structure). 
See https://developer.nrel.gov/docs/energy-optimization/reopt-v1/#ElectricTariff_collapsecontainer for more
